<launch>
    <param name="/use_sim_time" value="true"/>

    <!-- rviz -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find vineslam_ros)/param/config.rviz" required="false"/>

    <!-- agrob_map_transform gps node -->
    <node pkg="agrob_map_transform" type="agrob_gnss_transform_node" name="agrob_gnss_transform_node"
          clear_params="true">
        <rosparam command="load" file="$(find agrob_map_transform)/params/navsat_transform_template.yaml"/>

        <remap from="imu/data" to="/Dummy/IMU"/>
        <remap from="odometry/filtered" to="/vineslam/odom"/>
        <remap from="gps/fix" to="/fix"/>
    </node>

    <!-- replay node -->
    <arg name="bagfile_name" default="/home/andresaguiar/catkin_ws_agro_nav_tests/src/agro_nav_tests/bag_files/aveleda_2020-06-25-11-56-54.bag"/>
    <node pkg="vineslam_ros" type="replay_node" name="replay_node" output="screen">
        <rosparam command="load" subst_value="True" file="$(find vineslam_ros)/param/setup_replay.yaml"/>
    </node>

    <!-- image feature node -->
   <include file="$(find vfe)/launch/run.launch" />
</launch>
